src/frontend/src/lib/constants/test-ids.constants.ts

```diff
@@ -11,6 +12,7 @@ export const LOADER_MODAL = 'loader-modal';
 export const TOKEN_CARD = 'token-card';
 export const TOKEN_GROUP = 'token-group';
 export const TOKEN_BALANCE = 'token-balance';
+export const TOKEN_TOGGLE = 'token-toggle';
```

Comment:
```suggestion
export const MANAGE_TOKENS_MODAL_TOKEN_TOGGLE = 'manage-tokens-modal-token-toggle';
```

and put it together with `MANAGE_TOKEN_LIST_SAVE` in a separate session that pertains to MANAGE modal